src/MultiDropdown.js

```diff
@@ -3,19 +3,79 @@ import PropTypes from "prop-types";
 import * as countriesCities from "countries-cities";
 import "./MultiDropdown.css";
 import _ from "lodash";
+import CustomDropdown from "./CustomDropdown";
 
 // HINT User CustomDropdown.js here
 
 export default class MultiDropdown extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      country: null,
+      cities: null
+    };
+  }
+  selectCountries(country) {
+    this.setState({ country });
+    this.props.onChange({ country, cities: null });
+  }
+  selectGetCities(cities) {
```

Comment:
Why is this called `selectGetCities`? Doesn't make any sense.

Reply:
done